### PR TITLE
updated opensource section of Home page

### PR DESCRIPTION
**Pull request by ashuvssut**
adds to #495 
added jwoc crosswoc gssoc mwoc

converted all images in webp

_--I have participated in DWoC SWoC and MWoC_

### CODE REVIEW

File: src/pages/Home/sections/OpenSource/OpenSource.js [hunk 1]
```diff
@@ -4,41 +4,75 @@ import styles from './OpenSource.module.css';
...
-        },
-        {
-            coverImage: '/images/events/SWOC-logo.png',
-            eventName:'Script Winter of Code 2021',
-            altName:'SWOC Logo',
-            eventDescription: 'Script Winter of Code envisioned by the Script Foundation aims to bring students into the world of open source development.',
-            referencePage:'https://swoc.tech/index.html'
-        }
+      {
+        coverImage: "/images/events/SLOP.png",
```
Comment (smaranjitghose) [inline src/pages/Home/sections/OpenSource/OpenSource.js:8]: Convert these images into `webp`  format as well 😊

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: src/pages/Home/sections/OpenSource/OpenSource.js (hunk 1 above)
Comment (ashuvssut) [reply src/pages/Home/sections/OpenSource/OpenSource.js]: no it is already converted in my previous PR. I have merged it just now

**Comment by smaranjitghose** [comment]
@ashuvssut Could you please remove JWOC. Rest is fine

**Comment by ashuvssut** [comment]
I observed that there are 3 different folder containing assets like svg and png images. I think we should keep them under one assets folder. Should i put it in `/src/assets` folder? or should I put it in some other folder

**Comment by ashuvssut** [comment]
> @ashuvssut Could you please remove JWOC. Rest is fine

done

**Comment by smaranjitghose** [comment]
> I observed that there are 3 different folder containing assets like svg and png images. I think we should keep them under one assets folder. Should i put it in `/src/assets` folder? or should I put it in some other folder

Yes, we need the assets inside the src directory. Remove the one on the root level of the repository

**Comment by ashuvssut** [comment]
> @ashuvssut Could you please remove JWOC. Rest is fine

should I remove JWOC from readme.md and add other WoC (those which are now present in  Opensource.js)?

**Comment by smaranjitghose** [comment]
> > @ashuvssut Could you please remove JWOC. Rest is fine
> 
> @smaranjitghose should I remove JWOC from readme.md and add other WoC (those which are now present in Opensource.js)?

Ya you can remove JWOC from everywhere as this project is not there in that program

**Comment by ashuvssut** [comment]
After this gets merged...I will start rearranging the images and put them in src/assets folder

**Pull request merged**